src/makeInfinityAdder.js

```diff
@@ -1,10 +1,27 @@
 'use strict';
 
 /***
+ *
  * @return {function}
  */
+
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+
+  const adder = (arg) => {
+    if (arg === undefined) {
+      const sum = result;
+      result = 0;
+
+      return sum;
+    }
+
+    result += arg;
```

Comment:
It's great that you've used `+=` to add the argument to the `result`. This is a concise way to express the accumulation of the values.